sdk/tables/azure-data-tables/azure/data/tables/_table_batch.py

```diff
@@ -3,20 +3,25 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-from typing import Union, Any, Dict, Mapping, Optional, List, Tuple
+from typing import Union, Any, Mapping, Optional, List, Tuple, TYPE_CHECKING
 
 from azure.core import MatchConditions
-from azure.core.pipeline.transport import HttpRequest
 
 from ._common_conversion import _transform_patch_to_cosmos_post
 from ._models import UpdateMode, TransactionOperation
 from ._serialize import _add_entity_properties, _prepare_key, _get_match_condition
 from ._entity import TableEntity
-from ._generated import models, AzureTable
-from ._generated._configuration import AzureTableConfiguration
-from ._generated._serialization import Serializer, Deserializer
-from ._generated._vendor import prep_if_match
-
+from ._generated.operations._operations import (
+    build_table_insert_entity_request,
+    build_table_merge_entity_request,
+    build_table_update_entity_request,
+    build_table_delete_entity_request
+)
+
+if TYPE_CHECKING:
```

Comment:
nit: we import them without TYPE_CHECKING. I think we have talked about this when updating type hints to python3, it's not a big deal to import without TYPE_CHECKING.